.github/workflows/deploy.yml

```diff
@@ -41,19 +43,18 @@ jobs:
         with:
           python-version: "3.12"
       - name: Install dependencies
-        run: python -m pip install .[docs] --upgrade pip
+        run: |
+          python -m pip install .[docs] --upgrade pip
+          pip install sphinx sphinx-rtd-theme
```

Comment:
Thanks @geoffreyphipps-uw! I think we prefer the [furo](https://github.com/pradyunsg/furo) theme rather than the rtd theme. And it is include in the optional dependencies [`docs`](https://github.com/ihmeuw-msca/pypkg/blob/main/pyproject.toml#L24), and probably want to stick with that

Reply:
ok